docs/documentation/schedule/recent additions.md

```diff
@@ -34,88 +20,91 @@ Join the discussions on <a href="https://github.com/google/transit/pulls" target
 </div>
 <div class="row"></div> -->
 
-## Recently Merged Proposals &ensp;<img src="../../assets/pr-merged.svg" style="height:1em;"/>
+## Recently Merged Proposals &ensp;<img src="../../../assets/pr-merged.svg" style="height:1em;"/>
 
-Recently merged proposals that are now features of the [official GTFS Schedule specification](../reference). See the complete [Revision History](../process#revision-history) for more.
+Recently merged proposals that are now features of the [official GTFS Schedule specification](../reference). See the complete [Revision History](../../../community/spec_amendment_process/gtfs_schedule_amendment_process/#revision-history) for more.
 
 <div class="row">
     <div class="leftcontainer">
-        <h3 class="title"><a href="https://github.com/google/transit/pull/355" class="no-icon" target="_blank">Add fare media</a></h3>
-        <p class="maintainer">#355 by <a href="https://github.com/isabelle-dr" class="no-icon" target="_blank">isabelle-dr</a> was merged on Mar 14, 2023</p>
+        <h3 class="title"><a href="https://github.com/google/transit/pull/433" class="no-icon" target="_blank">Adopt GTFS Flex</a></h3>
```

Comment:
I added Flex as the latest change, is there anything that I might be missing? considering that this page only list significant changes @tzujenchanmbd

Reply:
I think everything else is up to date. Thanks for adding Flex!